docs/documentation.md

```diff
@@ -1 +1,69 @@
-# PURIS Frontend Documentation
\ No newline at end of file
+# PURIS Frontend Documentation
+
+## Frontend Installation
+
+### Requirements
+To use the PURIS frontend, you need:
+
+- a running [PURIS backend](https://github.com/eclipse-tractusx/puris-backend/)
+- a running [Product EDC](https://github.com/catenax-ng/product-edc)
```

Comment:
Please do not reference catenax-ng repositories. EDC is also available in [Eclipse Tractusx](https://github.com/eclipse-tractusx/tractusx-edc).

Reply:
We will update this in a newer version!